### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by ssandino**
- adding new owner for .md files
- @renestalder do you want to get these emails too?

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -9,7 +9,8 @@ firestore.rules andras@socialincome.org michael@socialincome.org
...
 storage.rules andras@socialincome.org michael@socialincome.org
 
-admin/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
-functions/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
-shared/ andras@socialincome.org michael@socialincome.org dev@socialincome.org
+admin/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
+functions/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
+shared/ andras@socialincome.org michael@socialincome.org sandino@socialincome.org
 
+.md/ sandino@socialincome.org andras@socialincome.org michael@socialincome.org
```
Comment (renestalder) [inline CODEOWNERS:16]: @ssandino That except a folder `.md/`.

I think you have to check the example here regarding file extensions.

**Comment by renestalder** [comment]
@ssandino 

> do you want to get these emails too?

I guess I would get them anyway in those folder e.g. ui where I'm owner and not for those where I'm not owner, so that's fine.

File: CODEOWNERS (hunk 1 above)
Comment (renestalder) [reply CODEOWNERS:16]: [do you want to get these emails too?](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)

**Comment by ssandino** [comment]
.md should do it, right? (without the slash). 

Based on the example: 

> Order is important; the last matching pattern takes the most
> # precedence. When someone opens a pull request that only
> # modifies JS files, only @js-owner and not the global
> # owner(s) will be requested for a review.
> *.js    @js-owner #This is an inline comment.

**Comment by renestalder** [comment]
Okay, let's test that and see if it works.

**Review by renestalder — APPROVED** [review]

**Comment by ssandino** [comment]
> Okay, let's test that and see if it works.

@renestalder works (see #228)

**Pull request merged**